src/components/10-atoms/input-text/index.js

```diff
@@ -244,11 +251,13 @@ class AXAInputText extends AXAPopupMixin(NoShadowDOM) {
       placeholder = '',
       disabled,
       isReact,
-      maxLength,
+      maxLength = '',
```

Comment:
@ REVIEWERS: minor bugfix - if this attribute is not set on component the DOM was `maxlength="undefined"`